docs/administrator_guide/trade_reconstruction.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: default
+title: Trade Reconstruction
+parent: User Guide
+nav_order: 9
+
+
+---
+
+# Trade Reconstruction
+
+{: .no_toc }
+
+Trade Reconstruction is used to automatically link trade and orders from Order Management Systems (OMS) or Trade Capture systems to their related communications. There are configurations to implement before Trade Reconstruction can be performed on a Trade.
+{: .fs-6 .fw-300 }
+
+1. TOC
+
+{:toc}
+
+---
+
+## Overview
+
+Each Trade object has a connected Trade Reconstruction Configuration, which is used to add the criteria necessary to connect Trades to their related Communications. Trade Reconstruction Configruations have two components: the Search Criteria field, which defines the parameters of the saved search created for the Trade, and the Trade Keywords Field, which defines the Trade Keywords created for the Trade. Below is how to create a Trade Reconstruction Configuration, details on the Search Criteria field for Trade Reconstruction Configurations, details about Trade Keywords, and the limitations of Trade Reconstruction.
+
+
+
+## Creating A Trade Reconstruction Configuration
+
+The Trade Reconstruction Configuration attached to your Trade is a key component to Trade Reconstruction, as it determines how Trades will be connected to their related communications. **This must be created before you can create your Trade**. To create a Trade Reconstruciton Configuration, follow the steps below:
+
+1. Naviagte to the `Trade Reconstruction Configurations` tab and click `New Trade Reconstruction Configuration`.
+
+   ![](C:\Users\jaime.becker\Desktop\trace\documentation\relativity-trace-documentation\docs\administrator_guide\media\trade_reconstruction\trade_reconstruction_configuration_layout.PNG)

```

Comment:
@jaimebecker Can you please add this image to the a folder within the repo and reference it correctly. I can't fix this issue on my own.

Reply:
Nevermind. I see it and updated how it's referenced.